pages/news/2020-XX-XX-concrete-action.html

```diff
@@ -0,0 +1,56 @@
+title: Concrete Action: Taking a Stand with Terminology and Recruitment
+author: Core Team, Edited
+date: 2020-06-08 01:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown:"extra" %}
+
+Alongside the protesters in the US and around the world, the Mixxx team unequivocally renounces police brutality.
```

Comment:
I feel like we could say a bit more here. For reference, this is what the Django project has on its website:

> The Django Software Foundation deeply values the diversity of our developers, users, and community. We are distraught by the suffering, oppression, and systemic racism the Black community faces every day. We can no longer remain silent.
>
> In silence, we are complicit. To eradicate racism, xenophobia, and all other forms of inequality, we must take action. It is our responsibility and our honour to use our platform to amplify the voices of the oppressed and marginalised.
> 
> We ask today—and always—that the Django community comes together in support of the Black community. They will be heard.
>
> #BlackLivesMatter

Reply:
Seconding the "silence is violence" concept.

Reply:
new opening graf added